package.json

```diff
@@ -56,7 +56,6 @@
         "regenerator-runtime": "^0.13.2",
         "sass-lint": "^1.12.1",
         "sass-loader": "^12.5.0",
-        "raw-loader": "^4.0.1",
```

Comment:
This was intented?

Reply:
No, not really sure what happened, c/p did it's own thing I guess. Amended again :)